README.md

```diff
@@ -164,4 +195,4 @@ explain why.
 
 ## A2.7
 
-«replace this with your answer»
+The simplest Chomsky grammar level is a type-2 grammar. The very possibility of having an empty terminal means that grammar cannot be a type-3 grammar and the context doesn't really matter so the similest possible grammar is a type-2 grammar.
```

Comment:
# 0

It's actually type 3, as it can be implemented by a state machine with no additional stack. You can have empty productions in a type three grammar (remember that the BNF is not necessarily the grammar)